lib/insights/api/common/rbac/validate_groups.rb

```diff
@@ -12,7 +12,7 @@ def process
 
             Service.call(RBACApiClient::GroupApi) do |api|
               uuids = SortedSet.new
-              Service.paginate(api, :list_groups, {}).each { |group| uuids << group.uuid }
+              Service.paginate(api, :list_groups, {:uuid => @group_uuids.to_a}).each { |group| uuids << group.uuid }
```

Comment:
@mkanoor what is `@group_uuids` at this point?  Does this need `to_a` or `Array(@group_uuids)`

Reply:
@gmcculloug Its an SortedSet 
https://github.com/RedHatInsights/catalog-api/blob/1b46ac5cd30dbb92ae53376f1bcffaa367f7787b/app/services/catalog/share_resource.rb#L4